config/environments/development.rb

```diff
@@ -20,7 +20,7 @@
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
-  config.assets.debug = false
+  config.assets.debug = true
```

Comment:
So "true" means "do not concatenate or preprocess", and "false" means "concatenate and preprocess"?  That's a confusingly named parameter...

Reply:
Yep!

Reply:
Actually, true means "debug", which means "do not concatenate or preprocess"